frontend/projects/abtesting/src/assets/i18n/en.json

```diff
@@ -131,6 +131,10 @@
   "home.view-experiment.ending-criteria.and.text": "and ",
   "home.view-experiment.ending-criteria.students.text": " student(s)  ",
   "home.view-experiment.ending-criteria.no-criteria.text": "No Ending Criteria",
+  "home.view-experiment.enrollment-start-time.title.text": "Start Time",
+  "home.view-experiment.enrollment-time-logs.title.text": "Enrollment Time Logs",
+  "home.view-experiment.enrollment-end-time.title.text": "End Time",
+  "home.view-experiment.enrollment-start-time-list.text": "Time Logs",
```

Comment:
Update key name to _home.view-experiment.enrollment-time-logs.text_ as start from the name doesn't used in the value.

Reply:
Code updated.